src/XIVLauncher.Core/CoreEnvironmentSettings.cs

```diff
@@ -39,6 +40,15 @@ public static string GetCleanEnvironmentVariable(string envvar, string badstring
         return string.Join(separator, Array.FindAll<string>(dirty.Split(separator, StringSplitOptions.RemoveEmptyEntries), s => !s.Contains(badstring)));
     }
 
+    public static int GetAltAppId(string? appid)
+    {
+        int result;
+        if (int.TryParse(appid, out result))
```

Comment:
You can use `if (int.TryParse(appid, out var result))` to make the code a bit more concise

Reply:
Alright, did this (with uint.TryParse() instead).